optuna/visualization/_hypervolume_history.py

```diff
@@ -0,0 +1,92 @@
+from typing import Sequence
+
+import numpy as np
+
+from optuna._experimental import experimental_func
+from optuna.study import Study
+from optuna.visualization._plotly_imports import _imports
+from optuna.visualization.matplotlib._hypervolume_history import _get_hypervolume_history_info
+from optuna.visualization.matplotlib._hypervolume_history import _HypervolumeHistoryInfo
```

Comment:
Could you move these to this file?

Reply:
I've moved these implementations to this file.
The corresponding test file has also moved from matplotlib_tests to visualization_tests.